apps/payments/next/app/[locale]/[offeringId]/[interval]/new/page.tsx

```diff
@@ -106,6 +107,21 @@ export default async function New({
       fxaUid,
     );
 
+    const updateTaxResult = await updateTaxAddressAction(
+      cart.id,
+      cart.version,
+      offeringId,
+      {
+        countryCode: taxAddress.countryCode,
+        postalCode: taxAddress.postalCode,
+      },
```

Comment:
Can we simply pass in `taxAddress` here, rather than recreating the object inline?